### ISSUE TITLE
BenchMark: CNN with 2 Conv Layers. Accuracy on FashionMNIST Dataset: 99.2% and on MNIST dataset: 99.1%

### ISSUE THREAD

**Issue by Noumanmufc1** [zalandoresearch/fashion-mnist#119]
The model details are as follows:

    Preprocessing by calculating mean and std beforehand
    Trained using Cross Entropy Loss and Adam Optimizer.

    The layers in sequence are:

    Convolutional layer with 6 feature maps of size 5 x 5
    BatchNorm layer followed by ReLU activation.
    Average Pooling layer of size 2 x 2.
    Convolutional layer with 16 feature maps of size 5 x 5
    BatchNorm layer followed by ReLU activation.
    Average Pooling layer of size 2 x 2.
    Fully connected layer of input size 400 and output size 120 followed by ReLU activation.
    Fully connected layer of input size 120 and output size 84 followed by ReLU activation.
    Fully connected layer of input size 84 and output size 10 .

Accuracy achieved on Fashion MNIST Test Dataset is 99.2 %
Accuracy achieved on MNIST Test Dataset is 99.1% .

This network has been implemented in PyTorch. The code can be found [here](https://gist.github.com/Noumanmufc1/60f00e434f0ce42b6f4826029737490a)

**Comment by hanxiao** [comment]
test accuracy of fashion-mnist at 99.2%??? very unlikely. please check your code.

**Comment by Noumanmufc1** [comment]
I did check it again. Its giving accuracy of 99.23% . I have attached screenshot of the output. You can check it manually too though.
![screenshot from 2018-07-17 09-39-26](https://user-images.githubusercontent.com/35970677/42796802-9979beba-89a5-11e8-8f58-7da9141d915a.png)

**Comment by hanxiao** [comment]
One common mistake is that people thought they are working with Fashion-MNIST, but they are actually working with MNIST. This mistake can be found repeatedly in #110 #47 #36 etc.

Our [benchmark table](https://github.com/zalandoresearch/fashion-mnist/issues/36) (contributed by many people) also suggests that it's impossible for simple CNN to get 99% test accuracy.

I'm sorry to say that, but it's highly likely due to some bug in your code. Please check your code again and again. Plotting those curves doesn't mean your data set is correct.

Here are two ways to check whether you are working with the correct dataset.

1. Check the MD5 of your dataset. And compare it with our [MD5 in the README.md](https://github.com/zalandoresearch/fashion-mnist#get-the-data). Details can be found [in this thread](https://github.com/zalandoresearch/fashion-mnist#get-the-data).
2. Plot your dataset, [as suggested here](https://github.com/mpekalski/zalando/blob/master/problem.ipynb)

Good luck!

**Comment by Noumanmufc1** [comment]
Oh I rechecked and there was some error in my dataset. Thanks for pointing it out. I have run the code again by downloading the dataset once again and now the accuracy is: 91.13. Is that okay? And should I open another issue for this accuracy or this will do fine? The code in the link is updated.
